lib/routes/users.js

```diff
@@ -2,11 +2,15 @@
  * Routes for user management for csfx-serv-withdraw
  * Contains implementation for user registration, update, etc
  */
-module.exports = function(config, AuthX, services)
-{
+module.exports = function (config, AuthX, services) {
   const express = require('express');
   const router = express.Router();
   const httpError = require('http-errors');
+  const nodemailer = require('nodemailer');
```

Comment:
Why we are using following npm module? Because sendgrid has it own npm module 
1.  "nodemailer": "^4.6.8",
2.   "nodemailer-sendgrid-transport": "^0.2.0",